src/App.tsx

```diff
@@ -1,17 +1,28 @@
-import React from 'react';
+import React, { useState, useMemo } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const filteredMovies = [...moviesFromServer];
+  const [query, getQuery] = useState('');
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
```
https://react.dev/reference/react/useState#usestate